cnd/src/http_api/route_factory.rs

```diff
@@ -98,19 +98,19 @@ pub fn create(
         .and(rfc003_facade)
         .and_then(http_api::routes::index::get_info);
 
-    let herc20_halight_bitcoin = warp::post()
-        .and(warp::path!("swaps" / "herc20" / "halight"))
+    let herc20_halbit = warp::post()
+        .and(warp::path!("swaps" / "herc20" / "halbit"))
         .and(warp::path::end())
         .and(warp::body::json())
         .and(facade.clone())
-        .and_then(http_api::routes::index::post_herc20_halight_bitcoin);
+        .and_then(http_api::routes::index::post_herc20_halbit);
 
-    let halight_bitcoin_herc20 = warp::post()
-        .and(warp::path!("swaps" / "halight" / "herc20"))
+    let halbit_herc20 = warp::post()
+        .and(warp::path!("swaps" / "halbit" / "herc20"))
```

Comment:
This is gonna break the e2e tests.

If you want, we can take this is an opportunity to talk about how we can make these kind of changes in a backwards compatible way and migrate clients over.

Reply:
For this change there is no reason to be backward compatible, the new routes are not live yet in any meaningful way. I agree we definitely will need to have this conversation at some stage though.

Reply:
For now I'm changing this back to "halight" and the e2e/SDK ticket can take responsibility for this one.

Reply:
The idea was to get them live with releasing soon... but we can also decide to postpone that after more cleanup. 
I just feel a release is needed so I feel that we actually did the job of supporting the 4 routes ;)

Reply:
Ok, if we are realeasing then we definitely want this in. Will add.

Reply:
I think it still makes sense to do this change in steps that are backwards compatible, i.e.:

- change everything inside cnd
- change the SDK to use the new paths, release a new version
- update to the new version of the SDK + change paths in cnd

Reply:
We cannot actually do the endpoint change in cnd and keep using the SDK in the tests anyways. This PR leaves the "halight" in the endpoint.

Reply:
Tracking issue: https://github.com/comit-network/comit-js-sdk/issues/253

Reply:
@thomaseizinger came up with a stroke of genius, we can just reach into the cnd.client member and use whatever enpoint we want :) Added 4 functions that fully control the endpoints we support.